### PR TITLE
doc(examples): add simple CDP example

### PR DESCRIPTION
**Pull request by stephenmathieson**
This patch adds an example of using axe-core with the Chrome Debugging Protocol. The example is very minimal and does not include all of axe-webdriverjs's features.

## Reviewer checks

**Required fields, to be filled out by PR reviewer(s)**
- [x] Follows the commit message policy, appropriate for next version
- [x] Has documentation updated, a DU ticket, or requires no documentation change
- [x] Includes new tests, or was unnecessary
- [x] Code is reviewed for security by: @isner

### CODE REVIEW

File: doc/examples/chrome-debugging-protocol/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# axe-chrome-debugging-protocol-example
+
+This (very minimal) example demonstrates how to use `axe-core` with the [Chrome Debugging Protocol](https://chromedevtools.github.io/devtools-protocol/).
+
+The example does not have feature pairty with [`axe-webdriverjs`](https://github.com/dequelabs/axe-webdriverjs), and does not run on `<iframe>`s.
```
Comment (marcysutton) [inline doc/examples/chrome-debugging-protocol/README.md]: Typo: parity

File: doc/examples/chrome-debugging-protocol/axe-cdp.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+			return new Promise((resolve, reject) => {
+				const axe = window.axe;
+				if (!axe) {
+					throw new Error('Unable to find axe-core');
+				}
+
+				// Finally, run axe-core
+				axe
+					.run()
+					// For some reason, when resolving with an object, CDP ignores
```
Comment (marcysutton) [inline doc/examples/chrome-debugging-protocol/axe-cdp.js]: Wow, what a gotcha. Thanks for commenting

File: doc/examples/chrome-debugging-protocol/README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+## To run the example
+
+- Ensure Node v8+ is installed and on `PATH`
+- Move to the `doc/examples/chrome-debugging-protocol` directory
+- Run `npm install`
+- Run `node axe-cdp.js http://www.deque.com` to run `axe-core` via Puppeteer against http://www.deque.com and output results to the terminal
```
Comment (marcysutton) [inline doc/examples/chrome-debugging-protocol/README.md]: I'm getting a conflicting port error even after a machine reboot:
```
Error running axe-core: connect ECONNREFUSED 127.0.0.1:9222
```

Comment (stephenmathieson) [reply doc/examples/chrome-debugging-protocol/README.md]: Good call, we need to start `google-chrome` with that port opened. Will update.

File: doc/examples/chrome-debugging-protocol/README.md (hunk 1 above)
Comment (stephenmathieson) [reply doc/examples/chrome-debugging-protocol/README.md]: Whoops. Will fix.

**Pull request merged**